### PR TITLE
refs #444 add comment about security/csrf component

### PR DESCRIPTION
**Pull request by steinkel**
Promote use of Security/Csrf components

### CODE REVIEW

File: src/Controller/AppController.php [hunk 1]
```diff
@@ -43,6 +43,10 @@ public function initialize()
 
         $this->loadComponent('RequestHandler');
         $this->loadComponent('Flash');
+        //Recommended security enhancements for your application
```
Comment (ravage84) [inline src/Controller/AppController.php]: For better readability, may be add a white line.

Please add a space affter ``//`` for comments.
For the commented out code, it's OK.

Comment (dereuromark) [inline src/Controller/AppController.php]: I agree:

    // See http...

Comment (inoas) [inline src/Controller/AppController.php]: In here it is different: https://github.com/cakephp/app/blob/master/config/bootstrap.php#L202-L205

Maybe /* Recommended security enhancements for your application */ ?

Comment (dereuromark) [inline src/Controller/AppController.php]: @ionas Those should only be used for multiline comments IMO

Comment (inoas) [inline src/Controller/AppController.php]: Don't agree, neither does this https://github.com/cakephp/app/blob/master/config/bootstrap.php#L168-L170

I'd say `/* foo */` is for inline comments whereas `//$code` would be for out-commented lines.

**Comment by bravo-kernel** [comment]
Instructions seems/felt interpretable to me. Maybe:

Uncomment settings to enable CakePHP recommended security settings. Or sth similar

**Comment by inoas** [comment]
`/* Enable following components for recommended CakePHP security settings. */`

**Comment by bravo-kernel** [comment]
👍🏼 Glad you could make some sense out of my drive-and-comment jibberish

**Pull request merged**